branching_basics2.R

```diff
@@ -0,0 +1,227 @@
+

```

Comment:
Not sure what this second file is for; I suspect it may need to be removed, if this is only for backup purpose.